src/components/atoms/deliveryOrderCard/DeliveryOrderCard.module.scss

```diff
@@ -0,0 +1,42 @@
+@import "../../../foundations/sass/all";
+
+.contact-card {
+  align-items: center;
+  background-color: $white-floral; // Add a background color to match your theme
```

Comment:
We consider creating a color variable for each component instead of using colors directly. Please create a color variable in foundations/sass/_colors.scss file and use it here. This will make it easier to switch colors in the future.